Fix: Removed upload csv buttons

sepehr2github opened the pull request:
## Description

- Remove import CSV buttons

Task: [ID]()

## Type of change

<!--- Please delete options that are not relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

Fabricevladimir reviewed (commented):
There's an issue with `MintTokenDescription` Component, can you do the link in html fix that James implemented?

sepehr2github commented:
> There's an issue with the `MintTokenDescription` Component, can you do the link in HTML fix that James implemented?

Thanks for mentioning, in new crowdin json I couldn't find any link regarding this so I removed the components and added a simple translation as a replacement

Fabricevladimir commented on packages/web-app/src/containers/actionBuilder/mintTokens/index.tsx:
```diff
@@ -343,30 +343,30 @@ export const MintTokenForm: React.FC<MintTokenFormProps> = ({
-      reader.readAsBinaryString(myFile);
-    }
-  };
+  // const handleCSVUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Funny thing is, I'm pretty sure this worked lol

Fabricevladimir reviewed (approved):
Thanks, Sepehr. Good to merge
